### DISCUSSION TITLE
How to use with wordpress ?

### DISCUSSION THREAD

**Discussion by khanakia** [FullHuman/purgecss#595] [General]
Hi i want to use purgecss with wordpress i saw they tutorial you have here https://purgecss.com/guides/wordpress.html#usage

But where do i add that code in wordpress ? 
There is no clear tutorial explaining how exactly use purgecss with wordpress.

**Comment by Chaostheorie** [comment]
This is, AFAIK, supposed to be used when working with stylesheets generated by WordPress for e.g. creating themes. The [list](https://github.com/FullHuman/purgecss/blob/master/packages/purgecss-with-wordpress/index.js) provided by `purgecss-with-wordpress` can be used to easily safe list the default CSS rules required by WordPress for the general layout. To apply a config, that employs `purgecss-with-wordpress`, you could use the purgecss CLI as part of your e.g. build process or CI/CD.

**Comment by Ffloriel** [comment]
Unfortunately, I'm not too familiar with WordPress development. But as far as I know, it's what @Chaostheorie said in the comment.  This should work like any other development setup, where you will need to call PurgeCSS through postCSS, or Webpack, or another build tool. I could find one reddit question about it with some suggestions [here](https://www.reddit.com/r/Wordpress/comments/ljqawn/using_tailwind_css_and_purge_css_with_wordpress/)
